README.md

```diff
@@ -82,7 +84,7 @@ The Web Clipper is a browser extension that allows you to save web pages and scr
 # Features
 
 - Desktop, mobile and terminal applications.
-- [Web Clipper](https://github.com/laurent22/joplin/blob/master/readme/clipper.md) for Firefox and Chrome.
+- [Web Clipper](https://github.com/laurent22/joplin/blob/master/readme/clipper.md) for Chrome.
```

Comment:
Removed Firefox until it's fixed and ready to go.

Reply:
Let's not remove this as we'll need to put it back when it's fixed. We have a pinned thread about it in the forum.